src/fuse/utils.rs

```diff
@@ -0,0 +1,76 @@
+macro_rules! impl_overflow_arithmetic {
+    ($target: ty) => {
+        impl OverflowArithmetic<$target> for $target {
+            #[inline]
+            fn overflow_add(self, other: $target) -> Self {
+                let (res, overflow) = self.overflowing_add(other);
+                debug_assert!(
+                    !overflow,
+                    "number = {} add number = {} overflowing",
```

Comment:
Better to print parameter type here so that it is easier to know which type is overflowed.

Reply:
like number = 5(u32) add number = XXX overflow ...

Reply:
Done.